components/bl-collapsiblePanel-component/src/index.js

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useMemo, useState } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function CollapsiblePanelComponent({ component, elRef, eventHandlers, pods }) {
+  const {
+    style, classList, display, width, titleOpen, titleDefault, background, color, iconColor, fontSize, padding
+  } = component;
+  const { onOpen, onClose } = eventHandlers;
+
+  const [active, setActive] = useState(null);
+
+  const panelContentPod = pods['panelContent'];
+
+  const titleStyles = { background, color, fontSize, padding };
+
+  const changeActive = () => setActive(!active);
+
+  useEffect(() => {
+    active ? onOpen() : active===false ? onClose() : null;
+  }, [active]);
+
+  const title = useMemo(() => {
+    if (!titleDefault) {
+      return;
+    }
+
+    return active ? titleOpen || titleDefault : titleDefault;
+  }, [active, titleOpen, titleDefault]);
+
+  component.show = () => setActive(true);
+  component.hide = () => setActive(false);
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div
+      className={ cn('bl-customComponent-collapsiblePanel', ...classList) }
+      style={{ ...style, width }}
+      ref={ elRef }>
+      <PanelTitle
+        styles={ titleStyles }
+        onClickFunction={ changeActive }
+        title={ title }
+        isActive={ active }
+        iconColor={ iconColor }
+      />
+      <div className={ cn('panel-content', { 'open': active, 'close': active === false }) }>
```

Comment:
First of all, !active is better than here active === false 

And the second:

```suggestion
      <div className={ cn('panel-content', active ? 'open' : 'close') }>
```